client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,70 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: white;
+  border-width: 0px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  border: 2px solid black;
+  border-radius: 8px;
+  border-collapse: separate;
+  border-spacing: 0;
+  display: block;
```

Comment:
You probably shouldn't be overriding the `display: table` type that table tags usually have. I think this may be why you're having a hard time with the auto sizing.

Reply:
Removing `display: block` results in
<img width="490" alt="Screen Shot 2022-12-16 at 2 26 08 PM" src="https://user-images.githubusercontent.com/63379656/208174007-d6428d5c-8381-45f6-a834-b2b7c7253d14.png">
So the autofill worked great, but a) there is random space around the table, and b) setting `--firstColWidth` in the browser debugger still has no effect.

Reply:
Which room should I be looking at to see what you're seeing? The tables in https://test.virtualtabletop.io/PR-1381/Roger don't have gaps around the table with `display: block` removed.

Reply:
Look at the lower right scoreboard there.

But - and this is very odd - I see different results locally than I do in the pr-test room, and the branch is up to date. The screen caps I posted were from my machine.

Reply:
Gotcha, so here's the bottom-right scoreboard with `display: block` removed:

<img width="260" alt="Screenshot 2022-12-16 at 5 38 20 PM" src="https://user-images.githubusercontent.com/366761/208200691-e7da1826-c5f7-470f-b5b1-487b50d2469c.png">

I think you should be moving the `overflow: auto` to the parent of the table, `.scoreboard`. You can tell it's not working right because the table is larger than its conainer. I don't think scrollable table elements work that well, but if the scroller is on the ancestor it renders correctly:

<img width="260" alt="Screenshot 2022-12-16 at 5 38 35 PM" src="https://user-images.githubusercontent.com/366761/208200581-bd23a823-c268-45ae-81b1-cff34282ac2b.png">

Then you'll also want to move the border to the `.scoreboard` div as well to fix the rendering of that:

<img width="260" alt="Screenshot 2022-12-16 at 5 39 59 PM" src="https://user-images.githubusercontent.com/366761/208200651-0c24b642-9fb4-47bc-ae32-a69877157078.png">

So the end css is:
```css
.scoreboard {
    border: 2px solid black;
    box-sizing: border-box;
    background: #fff;
    border-width: 2px;
    border-radius: 8px;
    display: flex;
    justify-content: center;
    align-items: center;
    overflow: auto;
}

.scoreboard table {
    border-collapse: separate;
    border-spacing: 0;
    height: 100%;
    width: 100%;
}
```

Reply:
So that mostly works. What I *really* wanted was for the width of the first column to be independently settable and the width of the remaining columns be split up using the remaining space. There is a CSS var `--firstColWidth` that is set on the scoreboard, as well as `--columns`. However, resetting `--firstColWidth` seems to have no effect. Once that's solved - in the past I tried using the formula `calc((100% - var(--firstColWidth)) / (var(--columns) - 1)` to compute the column widths, but somehow it was not accepted by the css processor.

Any ideas?

Reply:
Using `min-width: var(--firstColWidth)` seems to do the trick.

Reply:
I must be misunderstanding what you are saying. When I set `min-width` instead of `width` in `.scoreboard tbody tr td:nth-child(1)`, no setting in the css in JSON for the widget seems to actually change the width of the column. (Please feel free to commit css changes yourself if you'd like, BTW.)

Reply:
I'm pretty sure the commit I just made fixes it.  Use

```
 ".scoreboard td:first-child": {
      "min-width": "10px"
    },
```

in the scoreboard.  There is a minimum width below which it won't seem to go no matter what, but I can make it smaller than the other columns up to a certain point, or bigger than the other columns up to whatever I want, using this.  But please test.  I'm probably wrong.